client/src/components/App/Content/UsersManagement/useUsersManagement.ts

```diff
@@ -200,6 +200,22 @@ const useUsersManagement = ({ page, rowsPerPage, cellNameSort, setPage }: useUse
 
     const handleCloseDialog = () => setUserDialog({ isOpen: false, info: {} })
 
+    const setUserActivityStatus = (isActive: boolean, userId: string) : Promise<any> => {
+        const setUpdateActivityStatusLogger = logger.setup('GraphQL request to the DB');
```

Comment:
The flow is updating user status not the grapghql request

Reply:
to keep single source of truth